perun-openapi/openapi.yml

```diff
@@ -8205,6 +8205,21 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /urlinjsonout/usersManager/deleteUser:
+    post:
+      tags:
+        - UsersManager
+      operationId: deleteUser
+      summary: Deletes a user.
+      parameters:
+        - $ref: '#/components/parameters/userId'
+        - { name: force, description: "If true, use force deletion.", schema: { type: boolean },  in: query, required: false }
```

Comment:
It might be worth defining the force parameters in the parameters schema since we can reuse it in multiple methods (like the `userId` for example).